### PR TITLE
Expandable: Improve widget documentation readability

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] Documentation contained mixed HTML and JS code in examples,
        Method return types in signatures were missing
[Solution] Code examples were separated and missing return types were added

Signed-off-by: Pawel Kaczmarczyk <p.kaczmarczy@samsung.com>

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/602

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm [hunk 1]
```diff
@@ -221,10 +220,12 @@ <h3>Summary</h3>
    &lt;h6&gt;Expandable head&lt;/h6&gt;
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm]: add `<p>HTML code:</p>`

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm [hunk 2]
```diff
@@ -281,10 +282,12 @@ <h3>Summary</h3>
...
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm]: add `<p>HTML code:</p>`

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm [hunk 3]
```diff
@@ -277,14 +279,17 @@ <h3>Summary</h3>
                                 example:</p><p></p></span>
                             <pre name="code" class="examplecode
                             prettyprint">
+                            <p>HTML code:</p>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm]: Would you please re-check the location of this "\<p>HTML code:\</p>"?
It is included inside of \<pre>\</pre>, even it should be located outside of it.

Comment (Dominnio) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm]: Done

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm]: Done.

**Pull request merged**